target-gen/src/commands/elf.rs

```diff
@@ -106,6 +106,7 @@ pub fn cmd_elf(
                 flash_algorithms: vec![algorithm_name],
                 rtt_scan_ranges: None,
                 scan_chain: None,
+                default_binary_format: None,
```

Comment:
Deliberately setting these to `None` to avoid noise in generated yamls, very few targets will need to change this.